src/components/GracePeriodError/GracePeriodError.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { ReactComponent as AlertIcon } from 'assets/images/errors/alert.svg';
+import {
+  GracePeriodErrorStyled,
+  WarningMessageStyled,
+  IconStyled,
+  GracePeriodWrapperStyled
+} from './GracePeriodErrorStyled';
+
+const GracePeriodError = () => (
+  <GracePeriodWrapperStyled data-testid="grace-period-error">
+    <GracePeriodErrorStyled>
+      <IconStyled>
+        <AlertIcon data-testid="alert-svg" />
+      </IconStyled>
+      <p>
+        Your payment method has expired. Please{' '}
+        <WarningMessageStyled>make sure to update</WarningMessageStyled> it to
+        avoid losing access to your NFL+ subscription.
```

Comment:
Do we want to have **NFL+** for every client?

Reply:
no idea :(

Reply:
please add a translations here with a i18nKey. As a default message, we can add `it to avoid losing access to your subscription.` and clients will be able to adjust the copy for themselves